drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -7,7 +7,11 @@
  */
```

Comment:
add "for rf2x driver" in your commit title

Reply:
Done.